source/wp-content/themes/wporg-news-2021/sass/components/local-header/_local-header.scss

```diff
@@ -0,0 +1,50 @@
+.local-header {
+	--bar-min-height: 58px;
+	--bar-text-color: var(--wp--preset--color--white);
+	--bar-link-color: var(--wp--preset--color--white);
+	--bar-link-hover-color: var(--wp--preset--color--off-white-2);
+	--bar-background-color: var(--wp--preset--color--blue-1);
+
+	color: var(--bar-text-color);
+	min-height: var(--bar-min-height);
+	position: relative;
+	background: linear-gradient(to top, transparent var(--bar-min-height), var(--bar-background-color) var(--bar-min-height) 100%);
+
+	&::after {
+		content: "";
+		min-height: var(--bar-min-height);
+		position: absolute;
+		bottom: 0;
```

Comment:
```suggestion
		bottom: 1px;
```

Add the same fix here from #249, since I see the same 1px line in the mobile dropdown.

<img width="361" alt="Screen Shot 2022-01-28 at 11 37 34 AM" src="https://user-images.githubusercontent.com/541093/151585960-eb4cb553-5943-4ad9-9b42-f1bae4ea7068.png">